scripts/get-hashsets.js

```diff
@@ -1,19 +1,21 @@
 "use strict";
 
-const AppConstants = require("./app-constants").init();
-
 const fs = require("fs");
-const path = require("path");
 const request = require("request");
+const S3 = require("aws-sdk/clients/s3");
 
-const pkg = require("./package.json");
+const AppConstants = require("../app-constants").init();
```

Comment:
Not really related to anything, but depending on the merge order of this and #48, it looks like the other open PR removes the `AppConstants.init()` method:
https://github.com/mozilla/blurts-server/pull/48/files#diff-2f58926425d4d23092ed82739fda4cdeL25